docs/developer/decision-records/2022-10-31-aggregate-service-layer/README.md

```diff
@@ -0,0 +1,50 @@
+# Aggregate Service layer
+
+## Decision
+
+An "aggregate service" layer will be explicitly defined to provide a clear and simple interface to interact with the 
+entities in a safe way.
+
+Please note that in this document we're not referring to "services" as the components that are registered on the
+runtime to achieve extensibility.
+
+## Rationale
+
+For `AggregateService` we intend a component that provides a clear and simple interface to interact with entities (such as
+`Asset`s, `ContractNegotiation`s and so on) encapsulating some logic and features, like business validation, 
+transactional context, exception handling, ...
+
+Currently, they are located into the `data-management-api`. Moving them to an underlying layer will permit to other kind
+of API to being exposed over them, benefiting from their offered features.
+
+## Approach
+
+### SPI
+
+The service layer will be described by a generic `aggregate-service-spi` module that, in fact will contain only the
+`AggregateServiceResult` class, that represent a `Result` returned by an `AggregateService` method call.
```

Comment:
Here, you are referring to an `AggregateService`. Is that supposed to be an actual interface? If so, I would introduce it before using it.

Reply:
Forgot the `*`, because `AggregateService` is not an interface, is a suffix for the aggregate service interfaces.